src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -16,37 +66,110 @@ export class AppLauncherGitproviderService implements GitProviderService {
   connectGitHubAccount(redirectUrl: string): void {
     // let url = "https://github.com/login/oauth/authorize?client_id=" + this.clientId +
     //  "&redirect_uri=" + encodeURIComponent(redirectUrl);
-    this.redirectToAuth(redirectUrl);
+    this.link('https://github.com', redirectUrl);
+    //this.redirectToAuth(redirectUrl);
+  }
+
+  /**
+   * Get GitHub repos associated with given user name
+   *
+   * @param userName The GitHub user name
```

Comment:
`userName` is listed as a param for the function but is not in the function. Either the comment is wrong or the function is missing the param.